microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -107,18 +102,32 @@ The iOS simulator for Mac OS X comes with Xcode, which you [install from the App
 
 When you are done, learn how to work with the simulator through [Apple Developer documentation](https://help.apple.com/simulator/mac/current).
 
-> [!NOTE]
-> To avoid having to open Xcode every time you want to use the iOS Simulator, open <!--Xcode, or iOS Simulator?-->it, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
+To avoid having to open Xcode every time you want to use the iOS Simulator, open the iOS Simulator, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
+
+
+<!-- ------------------------------ -->
+#### Virtual machines
+
+To test your site on other operating systems, you can run virtual machines on your development device. For example, **Hyper-V** is a virtualization tool that lets you run various versions of Windows and Linux. To learn more, see [Introduction to Hyper-V on Windows 10](/virtualization/hyper-v-on-windows/about/).
+
+For other virtual machines, see [Virtual machines](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Testing_strategies#virtual_machines) in _Strategies for carrying out testing_ at MDN.
 
 
 <!-- ====================================================================== -->
 ## Cloud-based emulators and simulators
 
-If you aren't able to use the emulators and you don't have access to real devices, then cloud-based emulators are the next-best thing.  A big advantage of cloud-based emulators over real devices and local emulators is that you can automate unit tests for your site across different platforms.
+Cloud-based emulators and simulators are commercial solutions to test your website on different devices and browsers without installing anything on your development machine.  They are useful when you don't have access to real devices or when you can't use local emulators. Cloud-based solutions are also useful for automating tests across different platforms.
```

Comment:
deleting & moving this comment to right section: Cloud-based

a local commit that didn't get pushed yet, probably from a PB comment: integrate/check:

![image](https://github.com/user-attachments/assets/feeeaa73-f517-4e18-8103-660e4d730019)

If Device Emulation in Microsoft Edge DevTools isn't sufficient for your testing needs, you can test by using real devices instead.  If you don't have access to the real devices that you want to test on, setting up a virtual machine or using a cloud-based testing service can help.  A big advantage of cloud-based testing services is that they can help you automate testing for your site across different platforms and form factors.

@captainbrosset

Reply:
decided: keeping CB's draft, largely b/c it stays focused on 1 scenario instead of (per inherited from upstream) leading by negatively mentioning a different scenario:
Cloud-based emulators and simulators are commercial solutions to test your website on different devices and browsers without installing anything on your development machine.  They are useful when you don't have access to real devices or when you can't use local emulators. Cloud-based solutions are also useful for automating tests across different platforms.